ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplExternalTables.java

```diff
@@ -180,6 +195,14 @@ void dataLocationDump(Table table, FileList fileList, HiveConf conf)
       }
     }
 
+    void dbLocationDump(String dbName, Path dbLocation, FileList fileList,
+        HiveConf conf) throws Exception {
+      Path fullyQualifiedDataLocation = PathBuilder
+          .fullyQualifiedHDFSUri(dbLocation, FileSystem.get(hiveConf));
+      write(lineFor(dbName, fullyQualifiedDataLocation, hiveConf));
```

Comment:
lineFor assumes that the entry is for table name . We are passing here dbName. What are the side effect?

Reply:
Nopes, the name isn't used anywhere, it writes to `_external_tables_info` which is anyway going to get deleted